xt_training/utils/training.py

```diff
@@ -54,6 +54,8 @@ def train(args):
 
     device = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
     print('Running on device: {}'.format(device))
+    if loss_fn.weight:
```

Comment:
@DavidBegert this breaks for any loss function without a `weight` attribute. Should be changed to:
```python
if hasattr(loss_fn, 'weight') and loss_fn.weight:
```

Reply:
True